tools/picard/picard_AddCommentsToBam.xml

```diff
@@ -1,4 +1,4 @@
-<tool name="AddCommentsToBam" id="picard_AddCommentsToBam" version="1.126.0">
+<tool name="AddCommentsToBam" id="picard_AddCommentsToBam" version="1.135">
```

Comment:
See the bed tool, it uses `version="@WRAPPER_VERSION@.0"`, each tool includes an individual postfix'd number which can be updated individually of the whole set which should depend consistently on a single version of picard.